src/box/replication.cc

```diff
@@ -1059,7 +1062,10 @@ void
 replicaset_connect(const struct uri_set *uris,
 		   bool connect_quorum, bool keep_connect)
 {
+	struct replicaset_connect_state state;
+	memset(&state, 0, sizeof(state));
```

Comment:
The cond `wakeup` needs an explicit `fiber_cond_create()`. I see it is done below. Lets move it directly below this memset?

Reply:
My bad, fixed. The diff is in the other comment.